packages/parcel/src/parcel-run.ts

```diff
@@ -1,13 +1,20 @@
-import { ParcelOptions } from  'parcel-bundler';
 import { main } from '@effection/node';
 import { createParcelServer } from './parcel-server';
 import * as yargs from 'yargs';
 
+type ParcelOptions = {
+  [key: string]: unknown;
+};
+
 yargs
   .command('$0 [files..]', 'run the parcel server', x => x, (argv) => {
-    let options: ParcelOptions & { port?: number } = JSON.parse(argv.options as string);
+    let options: ParcelOptions = JSON.parse(argv.options as string);
+    let ctx = main(createParcelServer(argv.files as string[], options));
 
-    main(createParcelServer(argv.files as string[], options));
+    process.on('SIGTERM', () => {
```

Comment:
This should all be handled by the [`main()` method from `@effection/node`](https://github.com/thefrontside/effection/blob/master/packages/node/src/main.ts) which will halt the `createParcelServer` operation on both `SIGINT` _and_ `SIGTERM` If this isn't happening, then it's definitely a problem.